aerogear/mobile-security-service
# feat(ui): Add disable app modal (AEROGEAR-8469)

## Motivation
https://issues.jboss.org/browse/AEROGEAR-8469

## What
Adding view for app disable modal

## Why
Allows the user to disable all current versions for an app



## Verification Steps
1. Navigate to the app detailed page.
2. Click `Disable App`.
3. A modal will appear.
4. Enter text in the text field

## Checklist:


## Progress


## Additional Notes
